bridges/elrondToEth/steps/semiIntegrated_test.go

```diff
@@ -6,11 +6,11 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/ElrondNetwork/elrond-eth-bridge/bridges/bridge"
```

Comment:
to continue with the above mentioned suggestion:
`bridges/ethElrond/steps/elrondToEth/<steps>.go`
and for the other flow:
`bridges/ethElrond/steps/ethToElrond/<steps>.go`